### ISSUE TITLE
Protect

### ISSUE THREAD

**Issue by caberke** [mochman/MMM-Nest#16]
Should Protect appear if everything is normal?

(typo fixed)

**Comment by mochman** [comment]
spear?

**Comment by caberke** [comment]
i meant appear.... should Protect appear if everything is normal?

**Comment by mochman** [comment]
Yes, it should appear as a green circle around the temp display if you're using the `displaymode: "both"`. 
If you're using `displaymode: "protect"` you'll still see the green ring with nothing in the center.
